plugins/services/src/js/utils/__tests__/CreateServiceModalFormUtil-test.js

```diff
@@ -1,6 +1,27 @@
 const CreateServiceModalFormUtil = require('../CreateServiceModalFormUtil');
 
+const EMPTY_TYPES = [null, undefined, {}, [], ''];
```

Comment:
There's `0` (zero) as an empty value in the doc block, shall we include it as well?